ads/_config.js

```diff
@@ -175,6 +175,10 @@ export const adConfig = {
     prefetch: 'https://s.ato.mx/p.js',
   },
 
+  bidtellect: {
+    renderStartImplemented: true,
```

Comment:
I see you implement `render-start`, but when I test locally I am never able to receive it? Could you double check?